src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -13,6 +13,7 @@ describe('<PillButton> component', () => {
     const downArrowLabel = 'down arrow';
     const upArrowLabel = 'up arrow';
     const clearLabel = 'clear label';
+    const imgRole = 'img';
 
     let view: RenderResult;
 
```

Comment:
The line below (I cannot comment there) can be written simpler: `{ name: inputName }`

Reply:
:+1: